sros2/sros2/api/_keystore.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2020 Canonical Ltd
+# Copyright 2016-2019 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from cryptography import x509
+
+import lxml
+
+from sros2.policy import get_transport_default, get_transport_schema
+
+from . import _utilities
+
+
+_KS_CONTEXT = 'contexts'
+_KS_PUBLIC = 'public'
+_KS_PRIVATE = 'private'
+_DEFAULT_COMMON_NAME = 'sros2testCA'
+
+
+def create_keystore(keystore_path):
```

Comment:
*Outside the scope of this PR*:
Some of those functions wording are now unfortunate given that they live in `_keystore`, e.g. `_keystore.create_keystore`.